generators/base/templates/src/modules/App/module.js

```diff
@@ -0,0 +1,14 @@
+const initialState = {};
+
+// ACTION TYPES
+export const actionTypes = {};
```

Comment:
Actiontypes should be flow only, no const. Should also be in the flow folder as a lib